gzip_ecap_extended_compressible_types_v1_2.patch

```diff
@@ -0,0 +1,48 @@
+--- src/adapter_gzip.cc	Sun Jun 12 02:10:41 2016
++++ src/adapter_gzip.cc	Sun Jun 12 02:14:33 2016
+@@ -367,7 +367,8 @@
+ 
+ 	/**
+ 	 * Checks the Content-Type response header.
+-	 * At this time, only responses with "text/html" content-type are allowed to be compressed.
++	 * Added more real text types
++	 * Modified by Y.Voinov (c) 2016
+ 	 */
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256
 
Sure.

Here is corrected patch.


12.06.2016 18:04, Constantin Rack пишет:
>
> In gzip_ecap_extended_compressible_types_v1_2.patch
<https://github.com/c-rack/squid-ecap-gzip/pull/17#discussion_r66720017>:
>
> > @@ -0,0 +1,48 @@
> > +--- src/adapter_gzip.cc    Sun Jun 12 02:10:41 2016
> > ++++ src/adapter_gzip.cc    Sun Jun 12 02:14:33 2016
> > +@@ -367,7 +367,8 @@
> > +
> > +     /**
> > +      * Checks the Content-Type response header.
> > +-     * At this time, only responses with "text/html" content-type
are allowed to be compressed.
> > ++     * Added more real text types
> > ++     * Modified by Y.Voinov (c) 2016
> > +      */
>
> Could you please remove those 2 comment lines?
> This information is already in the git history after merging the PR.
> Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
<https://github.com/c-rack/squid-ecap-gzip/pull/17/files/50b61aad4922347d1e5509be78f044ea8fa1df55#r66720017>,
or mute the thread
<https://github.com/notifications/unsubscribe/AOUiqNTq5zVDJNFo21BY_UvhGxwZzQ3Bks5qK_W-gaJpZM4IzpAA>.
>

-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2
 
iQEcBAEBCAAGBQJXXVG2AAoJENNXIZxhPexG4/gIAMMpA9P+mIliXem/E2tjI4qJ
HW5kXqBzsQEyL/hl0cYsaxGUOgPhN7JHo6f3bLJpbDLdnI0sKwHevmGcS3g8UaJD
C0ax7n/W9z8qfsFb9zdPnXhTATU44x4DPMBDbIDbN6nfG2tzVUOhO1Jr/7sbhWYd
f1rcFD1euNuSYftDdZq8ico6kGOdnappAH6P88Sfqw65ZWP+XNzfp1n2TtT+3R2c
liBJangDv5Ya+yq6tS8tpFupDaq8MmAo8y5WMPNuCtwZ9WNbfNq9PncN9ljm5jvm
Oa4rgvcfzYhkOhbMcJlo99etbpkqS8ztQt44huG59+vDNW/O5ycTGFyQrJ9T9Kc=
=GROa
-----END PGP SIGNATURE-----